doghouse/server/ciutil/ciutil.go

```diff
@@ -0,0 +1,96 @@
+package ciutil
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/http"
+	"sync"
+)
+
+var (
+	muTravisIPAddrs sync.RWMutex
+	travisIPAddrs   = map[string]bool{}
+
+	// https://www.appveyor.com/docs/build-environment/#ip-addresses
+	appveyorIPAddrs = map[string]bool{
+		"74.205.54.20":    true,
+		"104.197.110.30":  true,
+		"104.197.145.181": true,
+		"146.148.85.29":   true,
+		"67.225.139.254":  true,
+		"67.225.138.82":   true,
+		"67.225.139.144":  true,
+		"138.91.141.243":  true,
+	}
+)
+
+// IsFromCI returns true if given request is from tructed CI provider.
+func IsFromCI(r *http.Request) bool {
+	return IsFromTravisCI(r) || IsFromAppveyor(r)
+}
+
+// IsFromTravisCI returns true if given request is from Travis CI.
+// https://docs.travis-ci.com/user/ip-addresses/
+func IsFromTravisCI(r *http.Request) bool {
+	muTravisIPAddrs.RLock()
+	defer muTravisIPAddrs.RUnlock()
+	return travisIPAddrs[ipFromReq(r)]
+}
+
+// https://docs.travis-ci.com/user/ip-addresses/
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported function UpdateTravisCIIPAddrs should be of the form "UpdateTravisCIIPAddrs ..."